packages/account-v2/src/pages/CurrencySelector/helpers.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import {
+    CurrencyAudIcon,
+    CurrencyBtcIcon,
+    CurrencyEthIcon,
+    CurrencyEurIcon,
+    CurrencyGbpIcon,
+    CurrencyLtcIcon,
+    CurrencyUsdcIcon,
+    CurrencyUsdIcon,
+    CurrencyUsdtIcon,
+} from '@deriv/quill-icons';
+
+export const CURRENCY_TYPES = {
+    CRYPTO: 'CRYPTO',
+    FIAT: 'FIAT',
+} as const;
+
+export function getCurrencyConfig(type: keyof typeof CURRENCY_TYPES) {
+    if (type === CURRENCY_TYPES.CRYPTO)
+        return [
+            {
+                icon: () => <CurrencyUsdIcon iconSize='lg' />,
+                key: 'USD',
+                title: 'US Dollar',
+            },
```

Comment:
can these be moved to constant file instead of a helper file?

Reply:
Will move these in the improvement PR 
Thanks @amina-deriv for the suggestion